### ISSUE TITLE
Problem with A4 size

### ISSUE THREAD

**Issue by rbuelund** [dompdf/dompdf#892]
Hi

I'm trying to create a pdf with a background image that fills the fill page. If I place a div with the dimensions: width:793px; height:1122px; - then it fills the whole page, but I also get a second page??
If I use the dimensions: 793px; height:1102px; - I only get one page - but a blank space of 20 px at the bottom - what is going on here ?? Also tried with width:595pt; height:841pt; - same problem

**Comment by bsweeney** [comment]
Which version of dompdf are you using? Do you have a more specific example?

What about absolutely positioning the div (`position: absolute; left: 0px; top: 0px;right: 0px; bottom: 0px;`).

**Comment by rbuelund** [comment]
Wau.. yes the position absolute did the trick - but why ?

**Comment by bsweeney** [comment]
Absolutely positioned content does not affect the "flow" of the document. It will be rendered only on the page where it is encountered.

This is something we should look further into, though.

**Comment by rbuelund** [comment]
Thank you for your help.

**Comment by luismabenitez** [comment]
Any update on this? Having same problem still with the absolute position:
        .black-wrapper{
            background-color:#323232;
            height:1102px;
            position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px;
        }

And having that white space of around 20px in the bottom.

**Comment by bsweeney** [comment]
@luismabenitez do you have a more complete example, including the Dompdf configuration? If you're using a DPI other than 72 that will impact how px measurements will translate to the PDF. Also, do you have any margin on the page?

**Comment by luismabenitez** [comment]
@bsweeney nope no margins and yes 72 or 96 having the exact same margins.

The page styles are these:
```
@page {
        size: a4 portrait;
        margin: 0px 0px 0px 0px !important;
        padding: 0px 0px 0px 0px !important;
}
```

and just in case: 
```
* { padding: 0; margin: 0; }
```

Then a page div with this:

```
<div class="black-wrapper">
</div>
```

where the black-wrapper class has this:
```
.black-wrapper{
        background-color:#323232;
        height:1102px;
        position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px;
}
```
<img width="819" alt="Captura de pantalla 2020-12-26 a las 13 01 01" src="https://user-images.githubusercontent.com/5616886/103151037-99fcdd00-477a-11eb-87f9-0f40bef9dd0f.png">

**Comment by bsweeney** [comment]
That's the render I get when the Dompdf DPI is set to 96. If I set it to 72 then the black-wrapper DIV [covers the page](http://eclecticgeek.com/dompdf/debug.php?identifier=c6b658a54bc5ec6bb94b5d726cce67c5).

**Comment by Mellthas** [comment]
The first sample works fine now. This should be addressed by 3cb71d24df0741a6826dcd3673accc2501343adf.
